kickoff/templates/includes/firefox_release.html

```diff
@@ -1,5 +1,12 @@
-<script type='text/javascript'>
+<script>
+
+partialsADIRelease={{ partialFXRelease|safe }};
+partialsADIBeta={{ partialFXBeta|safe }};
+partialsADIESR={{ partialFXESR|safe }};
```

Comment:
You can get rid of this entirely if you're pulling the ADI in JS, but it's worth noting that this only works because repr(array) in Python gives you something that JS can understand in some cases. Eg, it wouldn't work if you had Unicode strings (because they would repr as u'foo').